packages/ses-ava/src/command.js

```diff
@@ -0,0 +1,155 @@
+/* global process */
+/* eslint-disable no-await-in-loop, no-continue, no-labels, no-unreachable-loop */
+
+/* The ses-ava command allows a single package to run the same test suite with
+ * multiple configurations named in package.json under `sesAvaConfigs`.
+ * Each of these configurations requires a separate Ava config file like
+ * test/_ava.special.config.js and a name.
+ * The ses-ava command will by default run all tests in every mode but allows
+ * the user to pass --only-* and --no-* at any argument position for any of
+ * the named configurations to filter.
```

Comment:
Suggestion: rather than dynamically defining flags from configuration, use repeatable argument-bearing options.
```suggestion
 * The ses-ava command will by default run all tests in every mode but allows
 * the user to pass --only <name> and --exclude <name> (or shorthands -o <name>
 * and -x <name>) at any argument position for any of the named configurations
 * to filter.
```
(and if accepted, remember to update the documentation in e.g. README.md as well)

Reply:
I am sure I will regret this, but in the interest of using time efficiently, I’m going to merge without addressing this one, though it is absolutely a good idea.